package.json

```diff
@@ -52,6 +51,7 @@
     "mathjax-full": "^3.1.2",
     "ng2-semantic-ui": "github:venkateshwarans/ng2-semantic-ui#sourcing-circle",
     "ngx-bootstrap": "^5.6.2",
+    "font-awesome": "^4.7.0",
```

Comment:
@vaibhavbhuva is `font-awesome` required for QuML player?

Reply:
No, It's not required.